### ISSUE TITLE
Move primitive types into their own System.Windows.Primitives nuget package

### ISSUE THREAD

**Issue by AdmiralSnyder** [dotnet/wpf#6687]
Types like `System.Windows.Point`, that are not necessarily tied to WPF due to their inner structure, but merely logically, should be moved into a separate nuget package e.g. `System.Windows.Primitives` (just like there's `System.Drawing.Primitives`).

That way, it's possible to reference them from libraries without having to put `UseWPF` into the .csproj files (keeping them small, as e.g. test projects testing the algorithms apart form their UI implementation should be)

**Comment by lindexi** [comment]
@AdmiralSnyder Yeah, I do it in https://github.com/dotnet-campus/dotnetCampus.WPFType
